core-bundle/src/EventListener/DbafsMetadataSubscriber.php

```diff
@@ -57,6 +58,12 @@ public function enhanceMetadata(RetrieveDbafsMetadataEvent $event): void
             $event->set('importantPart', $importantPart);
         }
 
+        // Add subtitle information
+        if (!empty($language = $row['subtitlesLanguage'] ?? null)) {
+            $subtitle = new Subtitle($language, $row['subtitlesType'] ?? null);
+            $event->set('subtitles', $subtitle);
```

Comment:
Singular, maybe?

Reply:
No, that would be just one subtitle then. A .vtt file typically contains several entries, therefore "subtitles language" and "subtitles type" is correct.

Reply:
But the object currently only contains one of each. Is this supposed to be an array here, then?

Reply:
@m-vo I think what @leofeyer means is that in a video you don't have one "subtitle" - it's always called "subtitles" (because it's many words, not just one word 🙃). Thus may be the class and enum should also be renamed to `Subtitles` and `SubtitlesType`.

Reply:
I see. Here, the metadata is not for a "video" but for a single file, though. Having a plural key there would IMHO suggest a collection of elements: `item.extraMetadata.subtitle.type` vs `item.extraMetadata.subtitles.type`. 

Alternatively we could go with `SubtitleAttributes`.

Reply:
No, the point is the word "subtitle" is wrong in this context.

A "subtitle" is a secondary title to another heading. But in the context of videos it's always "subtitles", never _a_ subtitle.

(See also https://english.stackexchange.com/questions/526683/a-subtitle-vs-subtitles for instance)

Reply:
After rereading your comments I think I'm getting your point. You could argue that a french `subtitle` is a single line of the french `subtitles` for instance.

Ok, fine with me. But let's have the class and property please be in line.

Reply:
> A "subtitle" is a secondary title to another heading. But in the context of videos it's always "subtitles", never a subtitle.

According to wiktionary, it's the singluar form for both: https://en.wiktionary.org/wiki/subtitle

Reply:
Can we agree on both using `subtitles` as the key and `Subtitles` as the class name?

Reply:
> According to wiktionary, it's the singluar form for both: https://en.wiktionary.org/wiki/subtitle

That's just how wiktionary is structured. But in the context of video you never say "subtitle", it's always "subtitles".

Reply:
Done in 79379327d064375187b5495040d1aba849018ec9